content/05.conclusion.md

```diff
@@ -1,4 +1,19 @@
 ## Conclusion
-1. Summarize discussed positives and pitfalls
-2. Leave some open ended questions yet to be explored
-  1. Will come into play as I write this review paper
+
```

Comment:
It feels like many of the things in this section are restatements of what you've already said in the rest of the manuscript. Instead, could you write what the implications are?

I'm imagining it would start with:
* Knowledge graphs are being widely applied.
* Machine learning methods for automated extraction from text can make these more complete.
* Representing things in a low-dimensional space make these practical and new methods are being developed to best capture structure.

Then discuss in another paragraph:
* Evaluation is important to advances that move the field forward.
* Approaches to evaluation that can consider network biases will be important (cite xswap from @zietzm ).
* Better deployment of blinded literature review to avoid uncontrolled validation-by-google.

And perhaps finally close with:
* Likely important to capture networks with many node/edge types in the same space in a way that retains this complexity.
* Likely important to take a more sophisticated view of missingness as many different processes drive the gaps. (@jjc2718 has really crystalized this in my mind recently)

I think this has a lot of what you're saying here but it frames it differently than it is framed elsewhere in the paper. What's here right now is more of a restating in a very similar way to how you already covered it.